app/config/ApplicationConfig.scala

```diff
@@ -51,6 +51,8 @@ object ApplicationConfig extends ApplicationConfig with ServicesConfig {
   override lazy val callbackUrl = loadConfig("tamc.external-urls.callback-url")
   override lazy val ivNotAuthorisedUrl = loadConfig("tamc.external-urls.not-authorised-url")
 
+  def ggSignInUrl: String = "/gg/sign-in"
```

Comment:
could this not be a val

Reply:
it could but its a waste of memory the containers have plenty of processing power, but are limited of memory

Reply:
This is a variable though which should be a val. Does a def not require a memory allocation as a reference point?

Reply:
theyll both require reference point, the worst being lazy val which will require 2.
This stops storing the value from being stored.
unsure about the jvm..